### PR TITLE
chore: add 0.10.1 release docs

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
no

# Scope of work
Add docs for 0.10.1 release.

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/stories/releases/v0.10.1/v0.10.1.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# v0.10.1
+
+
+## 🐛 Fixes
+
+- SfHero: left arrow below image
+- SfHero: buttons not clickable fix
+- SfTextarea: change component from functional to standard to fix listeners bug 
+- SfCarousel: design broken 
+- SfArrow, AfCircleIcon: remove doubled listeners
```
Comment (justyna-13) [inline packages/vue/src/stories/releases/v0.10.1/v0.10.1.stories.mdx:14]: SfCircleIcon :)

Comment (AdamPawlinski) [reply packages/vue/src/stories/releases/v0.10.1/v0.10.1.stories.mdx]: fixed, thx

File: packages/vue/src/stories/releases/v0.10.1/v0.10.1.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## 🐛 Fixes
+
+- SfHero: left arrow below image
+- SfHero: buttons not clickable fix
+- SfTextarea: change component from functional to standard to fix listeners bug 
+- SfCarousel: design broken 
+- SfArrow, SfCircleIcon: remove doubled listeners
+- SfImage: no image on Storybook
```
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply packages/vue/src/stories/releases/v0.10.1/v0.10.1.stories.mdx:15]: ok I moved them.

File: packages/vue/src/stories/getting-started/pull-off-components.md [hunk 3]
```diff
@@ -11,15 +11,9 @@ With just one command you can have all components and their styles, so that no n
 
 ## Required
 
-<<<<<<< HEAD:packages/vue/src/stories/getting-started/pull-off-components.md
```
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply packages/vue/src/stories/getting-started/pull-off-components.md:14]: I pulled it from hotfix which came from master, so it should have this fix.

**Pull request merged**